### ISSUE TITLE
Copper low in CSF

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9759]
**Preferred term label:**
Copper low in cerebrospinal fluid

**Synonyms**
Be careful about the term "reduced" because that has redox implications.  note that "At 8 months of age, serum copper was in the normal range, but CSF copper was decreased".

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://www.omim.org/entry/620306

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
added as
HP:0034823
Decreased CSF copper concentration
